src/libcephsqlite.cc

```diff
@@ -0,0 +1,701 @@
+#include <iostream>
+#include <sstream>
+
+#include <pthread.h>
+#include <assert.h>
+#include <sqlite3.h>
+#include <unistd.h>
+#include <string.h>
+#include <stdio.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <limits.h>
+
+
+#include "include/rados/librados.hpp"
+#include "include/radosstriper/libradosstriper.hpp"
+#include "common/ceph_mutex.h"
+
+extern "C" {
```

Comment:
I believe most of these can be static. I don't think there is actually a need to mark them extern "C" either.